doc/explanation/storage.md

```diff
@@ -17,6 +17,7 @@ The following storage drivers are supported:
 - [ZFS - `zfs`](storage-zfs)
 - [Ceph RBD - `ceph`](storage-ceph)
 - [CephFS - `cephfs`](storage-cephfs)
+- [Ceph Object - `cephobject`](storage-cephobject)
 
 (storage-location)=
 ### Data storage location
```

Comment:
I thought about that, and I think merging is the way to go here. Probably will just label it "Ceph (all)" to cover all variants as otherwise the column header gets very long.

Reply:
Sounds good, yes.